sbi/samplers/ode_solvers/base.py

```diff
@@ -0,0 +1,64 @@
+from abc import abstractmethod
+from typing import Callable
+
+import torch.nn as nn
+from torch import Tensor
+from torch.distributions import Distribution
+from zuko.lazy import LazyDistribution
+
+NeuralODEFuncType = Callable[[Tensor, Tensor, Tensor], Tensor]
+
```

Comment:
This currently is not really used. You could add this is a `Protocol` i.e. see [here](https://github.com/StarostinV/sbi/blob/2a6f3016924843dca195460f56080b67c0090116/sbi/inference/potentials/base_potential.py#L85).